integration/experiment/system_characterization/run_aib.sh

```diff
@@ -5,16 +5,28 @@
 # Run application with GEOPM
 # Create a report
 
-if [[ $# -ne 1 ]]; then
-    echo "Usage: $0 <config_file>"
+set -x
+set -e
+
+if [[ $# -ne 2 ]]; then
+    echo "Usage: $0 <config_file> <intensity>"
     exit 1
 fi
 
 SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
 CONFIG_FILE=$1
-REGION=intensity_32
+INTENSITY=$2
+REGION=intensity_${INTENSITY}
 CORE_COUNT=$(geopmread -d | grep "core" | awk '{print $2}')
 RANK_COUNT=$((CORE_COUNT-4))
+
+
+if [[ ${INTENSITY} -gt 16 ]]; then
+    ITER_CNT=10
+else
+    ITER_CNT=20
+fi
```

Comment:
Comment here why this was done.  Re: memory intensity and noisy data.  Ensure this is all accurate.